Projects/hCoreUI/Sources/MarkdownText.swift

```diff
@@ -41,6 +41,7 @@ extension MarkdownText: Viewable {
         markdownText.numberOfLines = 0
         markdownText.lineBreakMode = .byWordWrapping
         markdownText.baselineAdjustment = .none
+        markdownText.isUserInteractionEnabled = true
```

Comment:
```suggestion
```

Reply:
I think you need this for links to be tappable.